### DISCUSSION TITLE
[4.0] Return the Print Function

### DISCUSSION THREAD

**Discussion by jweaver0312** [joomla/joomla-cms#35325] [2. Cold Features]
#35324

I think Joomla removing the print article feature specifically was a bad idea and that the print function should be returned. Even if it’s not brought into the core, at least make it an extension or a module.

**Comment by PhilETaylor** [comment]
https://extensions.joomla.org/extension/add-print-stylesheet/ and a load that convert to PDF https://extensions.joomla.org/tags/print-pdf/

**Reply by jweaver0312** [reply]
Add Print Stylesheet is not compatible with J4, even a forced install fails.

Most of them that serve that function are paid downloads with the only decent free one (Phone PDF) not installing on J4 despite claimed compatibility, albeit because it hasn't been updated since 2017. With most free ones not compatible with J4 at the same time. So in practicality, there's really nothing at the end of the day, with the print function's removal from the core.

Sure, now that its gone from the core, its only a matter of time before the extensions come up to replace it.

**Reply by PhilETaylor** [reply]
Do you actually realise that you can create a print Stylesheet without even having a plug-in? You know web development 101?

https://www.sitepoint.com/css-printer-friendly-pages/
